### ISSUE TITLE
The `ember-cli-version-checker` dependency is causing a deprecation: "An addon is trying to access project.nodeModulesPath"

### ISSUE THREAD

**Issue by lolmaus** [aexmachina/ember-cli-sass#174]
Happens in `ember-cli` `2.18.0-beta.1`.

> DEPRECATION: An addon is trying to access project.nodeModulesPath. This is not a reliable way to discover npm modules. Instead, consider doing: require("resolve").sync(something, { basedir: project.root }). Accessed from:   new NPMDependencyVersionChecker (/home/lolmaus/Code/ember-yuidoc-frontend/node_modules/ember-cli-sass/node_modules/ember-cli-version-checker/src/npm-dependency-version-checker.js:11:33)

Seems to be resolvable by upgrading `ember-cli-sass`'s `ember-cli`: 

https://github.com/ember-cli/ember-new-output/compare/v2.12.2...v2.18.0-beta.1#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R24

**Comment by aexmachina** [comment]
v7.1.2 updates to ember-cli 2.17, does that fix the problem?

**Comment by workmanw** [comment]
@aexmachina This is still an issue for us. Per @rwjblue addons should update to `"ember-cli-version-check": ^2.0.0`. See: https://github.com/ember-cli/ember-cli-version-checker/issues/48#issuecomment-354912902

**Comment by wilbyang** [comment]
do you guys know how to resolve this issue? should I use an older version of ember-cli?

**Comment by aexmachina** [comment]
This is resolved in v7.1.4

**Comment by aexmachina** [comment]
I'm getting failures on Travis, I'm guessing because ember-cli@2.18 uses Chrome headless rather than PhantomJS.

```
Error: Browser exited unexpectedly
            Non-zero exit code: null
            Stderr: 
             [0118/221612.122655:FATAL:setuid_sandbox_host.cc(157)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /opt/google/chrome/chrome-sandbox is owned by root and has mode 4755.
            Failed to generate minidump.
```

Any ideas? /cc @stefanpenner @rwjblue
